tests/hijack/db.js

```diff
@@ -1,534 +1,514 @@
 import { TestData } from '../_helpers/globals';
 import {
-  CleanTestData,
-  EnableTrackingMethods,
-  GetLastMethodEvents,
-  GetMeteorClient,
+  addAsyncTest,
+  callAsync,
+  dumpEvents,
+  getLastMethodEvents,
+  registerMethod,
   RegisterMethod
 } from '../_helpers/helpers';
+import assert from 'assert';
 
-Tinytest.add(
+addAsyncTest(
   'Database - insert',
-  function (test) {
-    EnableTrackingMethods();
-    let methodId = RegisterMethod(function () {
-      TestData.insert({aa: 10});
+  async function (test) {
+    const methodId = RegisterMethod(async function () {
+      await TestData.insertAsync({aa: 10});
       return 'insert';
     });
-    let client = GetMeteorClient();
-    client.call(methodId);
-    let events = GetLastMethodEvents([0, 2]);
-    let expected = [
-      ['start',undefined,{userId: null, params: '[]'}],
-      ['wait',undefined,{waitOn: []}],
-      ['db',undefined,{coll: 'tinytest-data', func: 'insert'}],
-      ['complete']
-    ];
-    test.equal(events, expected);
-    CleanTestData();
-  }
-);
 
-Tinytest.add(
-  'Database - insert with async callback',
-  function (test) {
-    EnableTrackingMethods();
-    let methodId = RegisterMethod(function () {
-      TestData.insert({aa: 10}, function () {
-        // body...
-      });
-      return 'insert';
-    });
-    let client = GetMeteorClient();
-    client.call(methodId);
-    let events = GetLastMethodEvents([0, 2]);
-    let expected = [
-      ['start',undefined,{userId: null, params: '[]'}],
-      ['wait',undefined,{waitOn: []}],
-      ['db',undefined,{coll: 'tinytest-data', func: 'insert', async: true}],
-      ['complete']
-    ];
-    test.equal(events, expected);
-    CleanTestData();
+    await callAsync(methodId);
+
+    let events = getLastMethodEvents([0, 2, 3]);
+
+    dumpEvents(events);
+
+    let expected = [['start',{userId: null,params: '[]'}],['wait',{waitOn: []}],['db',{coll: 'tinytest-data',func: 'insertAsync'}],['complete']];
```

Comment:
In this file a number of the expected arrays were moved to one line. Could you please change it back to the previous formatting with one line per event?